### PR TITLE
home page

### LINKED ISSUE #5: Front End - Homepage

### PR DESCRIPTION
**Pull request by Rosales-Karl-Nathan-BSCS12A1**
https://github.com/Sigma-Daddies/DocuVault-2.0/issues/5

### CODE REVIEW

File: DocuVault/HomePage.xaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      mc:Ignorable="d" 
+      Background="White"
+      Title="HomePage">
+
+    <Grid>
+        <!-- Define two columns: left for text, right for image -->
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="2*"/>
+            <!-- Left column (TextBlock) takes more space -->
+            <ColumnDefinition Width="1*"/>
```
Comment (Victoria-Jan-Paul-Andre-BSCS21A1) [inline DocuVault/HomePage.xaml:16]: ANO TO?

File: DocuVault/HomePage.xaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+﻿<Page x:Class="DocuVault.HomePage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+      xmlns:local="clr-namespace:DocuVault"
+      mc:Ignorable="d" 
+      Background="White"
+      Title="HomePage">
+
```
Comment (Victoria-Jan-Paul-Andre-BSCS21A1) [inline DocuVault/HomePage.xaml:10]: WHAT IS THIS?????????

File: DocuVault/HomePage.xaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+﻿<Page x:Class="DocuVault.HomePage"
```
Comment (Victoria-Jan-Paul-Andre-BSCS21A1) [inline DocuVault/HomePage.xaml:1]: MALI

**Review by Victoria-Jan-Paul-Andre-BSCS21A1 — CHANGES_REQUESTED** [review]
Paki ayos pla

**Review by Brave-Osio — APPROVED** [review]
okay

**Review by Victoria-Jan-Paul-Andre-BSCS21A1 — APPROVED** [review]
nice1

**Pull request merged**